tslearn/tests/test_matrixprofile.py

```diff
@@ -17,4 +17,19 @@ def test_consistent_woth_stumpy():
     X_tr = mp.fit_transform(X)
     X_tr_stumpy = stumpy.stump(X_stumpy, m=10)[:, 0].astype(np.float)
 
-    np.testing.assert_allclose(X_tr.ravel(), X_tr_stumpy)
\ No newline at end of file
+    np.testing.assert_allclose(X_tr.ravel(), X_tr_stumpy)
+
+
+def test_stumpy_wrapper():
```

Comment:
Maybe it could just be an extra check in the previous test no?

Reply:
Computing three times the matrix profile -- the original, the pure numpy version and the stumpy wrap -- and comparing the wrap with the original one, like this ? 
```
    mp = MatrixProfile(subsequence_length=10)
    mp_stumpy = MatrixProfile(subsequence_length=10, algorithm="stump")

    X_tr = mp.fit_transform(X)
    X_tr_stumpy_wrap = mp_stumpy.fit_transform(X)
    X_tr_stumpy = stumpy.stump(X_stumpy, m=10)[:, 0].astype(np.float)

    np.testing.assert_allclose(X_tr.ravel(), X_tr_stumpy)
    np.testing.assert_allclose(X_tr, X_tr_stumpy_wrap)
```

Reply:
Yep!